dspace/config/dspace.cfg

```diff
@@ -1579,6 +1579,12 @@ solr-database-resync.cron = 0 15 2 * * ?
 
 registration.verification.enabled = false
 
+#---------------------------------------------------------------#
```

Comment:
could you consider to add a new modules/google-captcha.cfg file? I think there are several configurations for Google Captcha. Perhaps it already deserves a single configuration file for it.

And also add a reference for it on `dspace.cfg`:
```
include = ${module_dir}/google-captcha.cfg
```